usmqe_tests/api/tendrl/README.rst

```diff
@@ -0,0 +1,3 @@
+There are 2 types of tests here:
```

Comment:
@mkudlej ou.. that's quite ugly :-/ it will be tricky for CI to launch it with the correct parameters...
So I'm voting for separated directory for common tests.